frontend/packages/process-editor/src/components/Canvas/CanvasActionMenu/CanvasActionMenu.tsx

```diff
@@ -21,33 +18,21 @@ export type CanvasActionMenuProps = {
  *
  * @returns {ReactNode} - The rendered component
  */
-export const CanvasActionMenu = ({
-  isEditorView,
-  onSave,
-  toggleViewModus,
-}: CanvasActionMenuProps): ReactNode => {
+export const CanvasActionMenu = ({ onSave }: CanvasActionMenuProps): ReactNode => {
   const { t } = useTranslation();
   const { numberOfUnsavedChanges } = useBpmnContext();
```

Comment:
Here, we would also need to have the ```isEditAllowed``` because we will only let the user be able to save if they have the rights to edit it (based on the appLibVersion).